libs/portals/admin/ids-admin/src/components/Application/Application.graphql

```diff
@@ -0,0 +1,70 @@
+# Get Application by ApplicationId and TenantId
+query GetApplicationsByApplicationId($input: ApplicationsInput!) {
+  authAdminApplications(input: $input) {
+    data {
+      applicationId
+      applicationType
+      availableEnvironments
+      environments {
+        id
+        environment
+        name
+        displayName {
+          locale
+          value
+        }
+        basicInfo {
+          applicationId
+          applicationSecret
+          idsURL
+          oAuthAuthorizationUrl
+          deviceAuthorizationUrl
+          oAuthTokenUrl
+          oAuthUserInfoUrl
+          openIdConfiguration
+          jsonWebKeySet
+        }
+        applicationUrls {
+          callbackUrls
+          logoutUrls
+          cors
+        }
+        lifeTime {
+          absoluteLifeTime
+          inactivityLifeTime
+          inactivityExpiration
+        }
+      }
+      defaultEnvironment {
+        id
+        environment
+        name
+        displayName {
+          locale
+          value
+        }
+        basicInfo {
+          applicationId
+          applicationSecret
+          idsURL
+          oAuthAuthorizationUrl
+          deviceAuthorizationUrl
+          oAuthTokenUrl
+          oAuthUserInfoUrl
+          openIdConfiguration
+          jsonWebKeySet
+        }
+        applicationUrls {
+          callbackUrls
+          logoutUrls
+          cors
+        }
+        lifeTime {
+          absoluteLifeTime
+          inactivityLifeTime
+          inactivityExpiration
+        }
+      }
+    }
```

Comment:
Should not query everything from both environments and defaultEnvironment. Maybe get `application.defaultEnvironment.environment` to know which one to show by default but I'd rather go with the first environment in the list and we make it consistent on the server.

Reply:
Solved in https://github.com/island-is/island.is/pull/10534